react/UploadQueue/Readme.md

```diff
@@ -10,7 +10,7 @@ the upload queue:
 * underneath the top-bar on mobile
 
 ```jsx
-import isTesting from '../helpers/isTesting'
+import isTesting from 'cozy-ui/transpiled/react/helpers/isTesting'
```

Comment:
Same, this helper is for cozy-ui only, it should not be exposed to the public API

Reply:
<img width="708" alt="Capture d’écran 2021-12-24 à 14 26 16" src="https://user-images.githubusercontent.com/8363334/147355927-bf5f15a0-2f25-4b65-937d-d5f1922419ad.png">

Reply:
Doing a project wide search&replace is to agressive here 😅 

This did unwanted changes. I'll point them out on new comments.